Tweet-311-Denver/tweet_311_denver_ui
# Add confer

## What does this PR do?
- Adds testing for the `welcome` page
- Adds the conformation ID to the `Success` page 
- Refactors where we call `setCase()` moving it to `Success`

## How to test
- Run `get fetch`
- `npm I`
- You can run `npm test` to see the coverage 
- Also you can check App state for the caseID

## Issues:
- This PR closes no issues
- The board is up to date 

## Notes:
- Make sure and give this a good look over before we merge it. Moving around the `setCase` might have caused some issues but I think it's fine. 

Thanks!
@cammac60
